### PR TITLE
Updated Android and Darwin OS maps; bumped gem version to 1.5.3

### PR DESCRIPTION
**Pull request by liambly**
- Updates the Android OS mapping: https://developer.android.com/studio/releases/platforms
- Updates the Darwin OS mappings: https://theapplewiki.com/wiki/Kernel (note that the [previous source](https://www.theiphonewiki.com/wiki/Kernel) now directs to this URL)
- Bumps the gem version to 1.5.3

### CODE REVIEW

**Review by johvet — APPROVED** [review]

**Review by ujasur — APPROVED** [review]

File: lib/user_agent/operating_systems/darwin.rb [hunk 1]
```diff
@@ -127,7 +134,14 @@ module Darwin
         '21.6.0' => '12.5',    # also 12.5.1, 12.6 and 12.6.1
         '22.0.0' => '13.0',
         '22.1.0' => '13.0',
```
Comment (hungyunliao) [inline lib/user_agent/operating_systems/darwin.rb]:
```suggestion
        '22.1.0' => '13.0',    # also 13.0.1
```

File: lib/user_agent/operating_systems/darwin.rb [hunk 2]
```diff
@@ -148,29 +162,36 @@ module Darwin
         '17.7.0' => '4.3.2',
         '18.0.0' => '5.0.x',
         '18.2.0' => '5.1.x',
-        '18.5.0' => '5.2.x', # also 5.3.x
-        '19.0.0' => '6.0.x', # also 6.1.
+        '18.5.0' => '5.2.x',  # also 5.3.x
+        '19.0.0' => '6.0.x',  # also 6.1.
```
Comment (hungyunliao) [inline lib/user_agent/operating_systems/darwin.rb]:
```suggestion
        '19.0.0' => '6.0.x',  # also 6.1
```

**Review by hungyunliao — APPROVED** [review]
Just two nits. Approving.

**Comment by skwongr** [comment]
This is good @liambly, thank you for updating! Don't forget to run the rake tasks to update the `dim_device_user_agents` after the new gem is deployed. In Prod, you would need to pause the ELT since the table is large.
`art19:redshift:dimensions:user_agents:list`
`art19:redshift:dimensions:user_agents:update[s3://path-to-file.jsonl]`

**Pull request merged**